frontend/src/Games/DTI48/dti48-upgrade-chain.ts

```diff
@@ -43,13 +36,13 @@ function getDirectChildrenForDTI48OrganizationTree<M extends SimplifiedMemberFor
       }
       return [allPMs[allPMs.findIndex((pm) => pm.netid === member.netid) - 1]];
     }
-    case 'lead':
+    case 'lead': {
       // Design Leads have no children :(
-      if (DESIGN_LEAD_NETIDS.includes(member.netid)) return [];
-      if (BUSINESS_LEAD_NETID === member.netid) {
+      if (member.subteams.includes('design-leads')) return [];
```

Comment:
Should we create constants for `design-leads`, `business-leads`, etc

Reply:
added